driver/web/adapter.go

```diff
@@ -74,7 +77,7 @@ func (we Adapter) Start() {
 	router := mux.NewRouter().StrictSlash(true)
 
 	// handle apis
-	mainRouter := router.PathPrefix("/notifications/api").Subrouter()
```

Comment:
This will break the current applications as you change the APIs. The APIs does not have to be modified within this issue.